  [1m[35m (637.5ms)[0m  [1m[35mCREATE DATABASE "IntegracaoAPI_development" ENCODING = 'unicode'[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (675.1ms)[0m  [1m[35mCREATE DATABASE "IntegracaoAPI_test" ENCODING = 'unicode'[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (125.6ms)[0m  [1m[35mCREATE TABLE "schema_migrations" ("version" character varying NOT NULL PRIMARY KEY)[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (112.6ms)[0m  [1m[35mCREATE TABLE "ar_internal_metadata" ("key" character varying NOT NULL PRIMARY KEY, "value" character varying, "created_at" timestamp NOT NULL, "updated_at" timestamp NOT NULL)[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (1.0ms)[0m  [1m[34mSELECT pg_try_advisory_lock(849118151704667670)[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ bin/rails:9
Migrating to CreateAgressors (20180926230045)
  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (124.9ms)[0m  [1m[35mCREATE TABLE "agressors" ("id" bigserial primary key, "nome" character varying, "latitude" float, "longitude" float, "created_at" timestamp NOT NULL, "updated_at" timestamp NOT NULL)[0m
  ‚Ü≥ db/migrate/20180926230045_create_agressors.rb:3
  [1m[36mActiveRecord::SchemaMigration Create (1.5ms)[0m  [1m[32mINSERT INTO "schema_migrations" ("version") VALUES ($1) RETURNING "version"[0m  [["version", "20180926230045"]]
  ‚Ü≥ bin/rails:9
  [1m[35m (18.0ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ bin/rails:9
  [1m[36mActiveRecord::InternalMetadata Load (1.0ms)[0m  [1m[34mSELECT  "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2[0m  [["key", "environment"], ["LIMIT", 1]]
  ‚Ü≥ bin/rails:9
  [1m[35m (0.3ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ bin/rails:9
  [1m[36mActiveRecord::InternalMetadata Create (0.8ms)[0m  [1m[32mINSERT INTO "ar_internal_metadata" ("key", "value", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "key"[0m  [["key", "environment"], ["value", "development"], ["created_at", "2018-09-26 23:00:59.130247"], ["updated_at", "2018-09-26 23:00:59.130247"]]
  ‚Ü≥ bin/rails:9
  [1m[35m (5.3ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.5ms)[0m  [1m[34mSELECT pg_advisory_unlock(849118151704667670)[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.3ms)[0m  [1m[34mSELECT pg_try_advisory_lock(849118151704667670)[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ bin/rails:9
Migrating to CreateVitimas (20180926230138)
  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (109.0ms)[0m  [1m[35mCREATE TABLE "vitimas" ("id" bigserial primary key, "nome" character varying, "latitude" float, "longitude" float, "agressor_id" bigint, "created_at" timestamp NOT NULL, "updated_at" timestamp NOT NULL, CONSTRAINT "fk_rails_9974f45197"
FOREIGN KEY ("agressor_id")
  REFERENCES "agressors" ("id")
)[0m
  ‚Ü≥ db/migrate/20180926230138_create_vitimas.rb:3
  [1m[35m (57.0ms)[0m  [1m[35mCREATE  INDEX  "index_vitimas_on_agressor_id" ON "vitimas"  ("agressor_id")[0m
  ‚Ü≥ db/migrate/20180926230138_create_vitimas.rb:3
  [1m[36mActiveRecord::SchemaMigration Create (1.5ms)[0m  [1m[32mINSERT INTO "schema_migrations" ("version") VALUES ($1) RETURNING "version"[0m  [["version", "20180926230138"]]
  ‚Ü≥ bin/rails:9
  [1m[35m (23.2ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ bin/rails:9
  [1m[36mActiveRecord::InternalMetadata Load (1.9ms)[0m  [1m[34mSELECT  "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2[0m  [["key", "environment"], ["LIMIT", 1]]
  ‚Ü≥ bin/rails:9
  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.2ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.3ms)[0m  [1m[34mSELECT pg_advisory_unlock(849118151704667670)[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.3ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.9ms)[0m  [1m[35mBEGIN[0m
  [1m[36mAgressor Create (1.2ms)[0m  [1m[32mINSERT INTO "agressors" ("nome", "created_at", "updated_at") VALUES ($1, $2, $3) RETURNING "id"[0m  [["nome", "agenor"], ["created_at", "2018-09-26 23:02:51.848250"], ["updated_at", "2018-09-26 23:02:51.848250"]]
  [1m[35m (19.0ms)[0m  [1m[35mCOMMIT[0m
Started POST "/api/v1/" for 127.0.0.1 at 2018-09-26 20:11:43 -0300
  
PG::ConnectionBad (FATAL:  autentica√ß√£o do tipo password falhou para usu√°rio "postgres"
FATAL:  autentica√ß√£o do tipo password falhou para usu√°rio "postgres"
):
  
pg (1.1.3) lib/pg.rb:56:in `initialize'
pg (1.1.3) lib/pg.rb:56:in `new'
pg (1.1.3) lib/pg.rb:56:in `connect'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:684:in `connect'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:215:in `initialize'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:40:in `new'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:40:in `postgresql_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:809:in `new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:853:in `checkout_new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:832:in `try_to_checkout_new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:793:in `acquire_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:521:in `checkout'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:380:in `connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1008:in `retrieve_connection'
activerecord (5.2.1) lib/active_record/connection_handling.rb:118:in `retrieve_connection'
activerecord (5.2.1) lib/active_record/connection_handling.rb:90:in `connection'
activerecord (5.2.1) lib/active_record/migration.rb:554:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
activesupport (5.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (5.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:61:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.1) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.1) lib/rails/engine.rb:524:in `call'
puma (3.12.0) lib/puma/configuration.rb:225:in `call'
puma (3.12.0) lib/puma/server.rb:658:in `handle_request'
puma (3.12.0) lib/puma/server.rb:472:in `process_client'
puma (3.12.0) lib/puma/server.rb:332:in `block in run'
puma (3.12.0) lib/puma/thread_pool.rb:133:in `block in spawn_thread'
Started POST "/api/v1/" for 127.0.0.1 at 2018-09-26 20:12:22 -0300
  
PG::ConnectionBad (FATAL:  autentica√ß√£o do tipo password falhou para usu√°rio "postgres"
FATAL:  autentica√ß√£o do tipo password falhou para usu√°rio "postgres"
):
  
pg (1.1.3) lib/pg.rb:56:in `initialize'
pg (1.1.3) lib/pg.rb:56:in `new'
pg (1.1.3) lib/pg.rb:56:in `connect'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:684:in `connect'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:215:in `initialize'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:40:in `new'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:40:in `postgresql_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:809:in `new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:853:in `checkout_new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:832:in `try_to_checkout_new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:793:in `acquire_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:521:in `checkout'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:380:in `connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1008:in `retrieve_connection'
activerecord (5.2.1) lib/active_record/connection_handling.rb:118:in `retrieve_connection'
activerecord (5.2.1) lib/active_record/connection_handling.rb:90:in `connection'
activerecord (5.2.1) lib/active_record/migration.rb:554:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
activesupport (5.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (5.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:61:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.1) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.1) lib/rails/engine.rb:524:in `call'
puma (3.12.0) lib/puma/configuration.rb:225:in `call'
puma (3.12.0) lib/puma/server.rb:658:in `handle_request'
puma (3.12.0) lib/puma/server.rb:472:in `process_client'
puma (3.12.0) lib/puma/server.rb:332:in `block in run'
puma (3.12.0) lib/puma/thread_pool.rb:133:in `block in spawn_thread'
Started POST "/api/v1/" for 127.0.0.1 at 2018-09-26 20:16:14 -0300
  
PG::ConnectionBad (FATAL:  autentica√ß√£o do tipo password falhou para usu√°rio "postgres"
FATAL:  autentica√ß√£o do tipo password falhou para usu√°rio "postgres"
):
  
pg (1.1.3) lib/pg.rb:56:in `initialize'
pg (1.1.3) lib/pg.rb:56:in `new'
pg (1.1.3) lib/pg.rb:56:in `connect'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:684:in `connect'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:215:in `initialize'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:40:in `new'
activerecord (5.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:40:in `postgresql_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:809:in `new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:853:in `checkout_new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:832:in `try_to_checkout_new_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:793:in `acquire_connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:521:in `checkout'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:380:in `connection'
activerecord (5.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1008:in `retrieve_connection'
activerecord (5.2.1) lib/active_record/connection_handling.rb:118:in `retrieve_connection'
activerecord (5.2.1) lib/active_record/connection_handling.rb:90:in `connection'
activerecord (5.2.1) lib/active_record/migration.rb:554:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
activesupport (5.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (5.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:61:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.1) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.1) lib/rails/engine.rb:524:in `call'
puma (3.12.0) lib/puma/configuration.rb:225:in `call'
puma (3.12.0) lib/puma/server.rb:658:in `handle_request'
puma (3.12.0) lib/puma/server.rb:472:in `process_client'
puma (3.12.0) lib/puma/server.rb:332:in `block in run'
puma (3.12.0) lib/puma/thread_pool.rb:133:in `block in spawn_thread'
Started POST "/api/v1/" for 127.0.0.1 at 2018-09-26 20:16:46 -0300
  [1m[35m (0.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ /home/matheusdalagnol/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/activerecord-5.2.1/lib/active_record/log_subscriber.rb:98
  
ActionController::RoutingError (No route matches [POST] "/api/v1"):
  
actionpack (5.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:65:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.1) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.1) lib/rails/engine.rb:524:in `call'
puma (3.12.0) lib/puma/configuration.rb:225:in `call'
puma (3.12.0) lib/puma/server.rb:658:in `handle_request'
puma (3.12.0) lib/puma/server.rb:472:in `process_client'
puma (3.12.0) lib/puma/server.rb:332:in `block in run'
puma (3.12.0) lib/puma/thread_pool.rb:133:in `block in spawn_thread'
Started POST "/api/v1/" for 127.0.0.1 at 2018-09-26 20:18:17 -0300
  
ActionController::RoutingError (No route matches [POST] "/api/v1"):
  
actionpack (5.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:65:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.1) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.1) lib/rails/engine.rb:524:in `call'
puma (3.12.0) lib/puma/configuration.rb:225:in `call'
puma (3.12.0) lib/puma/server.rb:658:in `handle_request'
puma (3.12.0) lib/puma/server.rb:472:in `process_client'
puma (3.12.0) lib/puma/server.rb:332:in `block in run'
puma (3.12.0) lib/puma/thread_pool.rb:133:in `block in spawn_thread'
Started POST "/api/v1/" for 127.0.0.1 at 2018-09-26 20:18:41 -0300
  
ActionController::RoutingError (No route matches [POST] "/api/v1"):
  
actionpack (5.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:65:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.1) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.1) lib/rails/engine.rb:524:in `call'
puma (3.12.0) lib/puma/configuration.rb:225:in `call'
puma (3.12.0) lib/puma/server.rb:658:in `handle_request'
puma (3.12.0) lib/puma/server.rb:472:in `process_client'
puma (3.12.0) lib/puma/server.rb:332:in `block in run'
puma (3.12.0) lib/puma/thread_pool.rb:133:in `block in spawn_thread'
Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-26 20:19:18 -0300
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"74.4568", "longitude"=>"77.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"74.4568", "longitude"=>"77.9865"}}
Completed 400 Bad Request in 0ms (ActiveRecord: 0.0ms)


  
ActionController::ParameterMissing (param is missing or the value is empty: agressor):
  
app/controllers/api/v1/posicao_agressor_controller.rb:28:in `agressor_params'
app/controllers/api/v1/posicao_agressor_controller.rb:4:in `nova_posicao'
Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-26 20:19:59 -0300
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"74.4568", "longitude"=>"77.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"74.4568", "longitude"=>"77.9865"}}
Unpermitted parameter: :posicao_agressor
Completed 500 Internal Server Error in 1ms (ActiveRecord: 0.0ms)


  
NoMethodError (undefined method `latitude=' for nil:NilClass):
  
app/controllers/api/v1/posicao_agressor_controller.rb:4:in `nova_posicao'
  [1m[36mAgressor Load (0.4ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" LIMIT $1[0m  [["LIMIT", 11]]
Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-26 20:21:47 -0300
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"74.4568", "longitude"=>"77.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"74.4568", "longitude"=>"77.9865"}}
  [1m[36mAgressor Load (0.3ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" WHERE "agressors"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:22
Unpermitted parameter: :posicao_agressor
Unpermitted parameter: :posicao_agressor
  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[36mAgressor Update (0.4ms)[0m  [1m[33mUPDATE "agressors" SET "latitude" = $1, "longitude" = $2, "updated_at" = $3 WHERE "agressors"."id" = $4[0m  [["latitude", 74.4568], ["longitude", 77.9865], ["updated_at", "2018-09-26 23:21:47.348475"], ["id", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[35m (19.6ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
Completed 500 Internal Server Error in 40ms (ActiveRecord: 25.3ms)


  
ArgumentError (wrong number of arguments (given 3, expected 0..2)):
  
app/controllers/api/v1/posicao_agressor_controller.rb:10:in `nova_posicao'
  [1m[36mAgressor Load (0.3ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" LIMIT $1[0m  [["LIMIT", 11]]
Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-26 20:23:34 -0300
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"74.4568", "longitude"=>"77.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"74.4568", "longitude"=>"77.9865"}}
  [1m[36mAgressor Load (0.4ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" WHERE "agressors"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:22
Unpermitted parameter: :posicao_agressor
Unpermitted parameter: :posicao_agressor
  [1m[35m (0.3ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[35m (0.2ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
Completed 200 OK in 11ms (Views: 0.4ms | ActiveRecord: 3.8ms)


Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-26 20:23:51 -0300
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"88.4568", "longitude"=>"-7.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"88.4568", "longitude"=>"-7.9865"}}
  [1m[36mAgressor Load (0.4ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" WHERE "agressors"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:22
Unpermitted parameter: :posicao_agressor
Unpermitted parameter: :posicao_agressor
  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[36mAgressor Update (0.3ms)[0m  [1m[33mUPDATE "agressors" SET "latitude" = $1, "longitude" = $2, "updated_at" = $3 WHERE "agressors"."id" = $4[0m  [["latitude", 88.4568], ["longitude", -7.9865], ["updated_at", "2018-09-26 23:23:51.973789"], ["id", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[35m (18.3ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
Completed 200 OK in 22ms (Views: 0.3ms | ActiveRecord: 19.1ms)


  [1m[36mAgressor Load (0.3ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" ORDER BY "agressors"."id" DESC LIMIT $1[0m  [["LIMIT", 1]]
Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-26 20:24:39 -0300
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"88.4568", "longitude"=>"-7.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"88.4568", "longitude"=>"-7.9865"}}
  [1m[36mAgressor Load (0.3ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" WHERE "agressors"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:22
Unpermitted parameter: :posicao_agressor
Unpermitted parameter: :posicao_agressor
  [1m[35m (0.3ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[35m (0.2ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
Completed 200 OK in 3ms (Views: 0.2ms | ActiveRecord: 0.8ms)


Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-26 20:25:18 -0300
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"-17.4568", "longitude"=>"-7.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"-17.4568", "longitude"=>"-7.9865"}}
  [1m[36mAgressor Load (0.5ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" WHERE "agressors"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:22
Unpermitted parameter: :posicao_agressor
Unpermitted parameter: :posicao_agressor
  [1m[35m (0.3ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[36mAgressor Update (0.5ms)[0m  [1m[33mUPDATE "agressors" SET "latitude" = $1, "updated_at" = $2 WHERE "agressors"."id" = $3[0m  [["latitude", -17.4568], ["updated_at", "2018-09-26 23:25:18.348707"], ["id", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[35m (16.2ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
Completed 200 OK in 21ms (Views: 0.4ms | ActiveRecord: 17.5ms)


Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-26 20:26:56 -0300
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"-17.4568", "longitude"=>"-7.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"-17.4568", "longitude"=>"-7.9865"}}
  [1m[36mAgressor Load (0.3ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" WHERE "agressors"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:19
Unpermitted parameter: :posicao_agressor
Unpermitted parameter: :posicao_agressor
  [1m[35m (0.1ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[35m (0.1ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
Completed 200 OK in 11ms (Views: 0.1ms | ActiveRecord: 3.4ms)


  [1m[35m (0.3ms)[0m  [1m[34mSELECT pg_try_advisory_lock(849118151704667670)[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (29.3ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ bin/rails:9
  [1m[36mActiveRecord::InternalMetadata Load (10.0ms)[0m  [1m[34mSELECT  "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2[0m  [["key", "environment"], ["LIMIT", 1]]
  ‚Ü≥ bin/rails:9
  [1m[35m (0.4ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.3ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.4ms)[0m  [1m[34mSELECT pg_advisory_unlock(849118151704667670)[0m
  ‚Ü≥ bin/rails:9
  [1m[35m (0.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ bin/rails:9
Started GET "/" for 127.0.0.1 at 2018-09-30 20:01:51 -0300
  [1m[35m (0.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ /home/matheusdalagnol/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/activerecord-5.2.1/lib/active_record/log_subscriber.rb:98
Processing by Rails::WelcomeController#index as HTML
  Rendering /home/matheusdalagnol/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/railties-5.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /home/matheusdalagnol/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/railties-5.2.1/lib/rails/templates/rails/welcome/index.html.erb (16.6ms)
Completed 200 OK in 85ms (Views: 60.0ms | ActiveRecord: 0.0ms)


Started GET "/favicon.ico" for 127.0.0.1 at 2018-09-30 20:01:51 -0300
  
ActionController::RoutingError (No route matches [GET] "/favicon.ico"):
  
actionpack (5.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:65:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.1) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.1) lib/rails/engine.rb:524:in `call'
puma (3.12.0) lib/puma/configuration.rb:225:in `call'
puma (3.12.0) lib/puma/server.rb:658:in `handle_request'
puma (3.12.0) lib/puma/server.rb:472:in `process_client'
puma (3.12.0) lib/puma/server.rb:332:in `block in run'
puma (3.12.0) lib/puma/thread_pool.rb:133:in `block in spawn_thread'
Started GET "/favicon.ico" for 127.0.0.1 at 2018-09-30 20:01:51 -0300
  
ActionController::RoutingError (No route matches [GET] "/favicon.ico"):
  
actionpack (5.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:65:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.1) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.1) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.1) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.1) lib/rails/engine.rb:524:in `call'
puma (3.12.0) lib/puma/configuration.rb:225:in `call'
puma (3.12.0) lib/puma/server.rb:658:in `handle_request'
puma (3.12.0) lib/puma/server.rb:472:in `process_client'
puma (3.12.0) lib/puma/server.rb:332:in `block in run'
puma (3.12.0) lib/puma/thread_pool.rb:133:in `block in spawn_thread'
Started POST "/api/v1/agressors/nova_posicao" for 127.0.0.1 at 2018-09-30 20:02:33 -0300
  [1m[35m (0.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  ‚Ü≥ /home/matheusdalagnol/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/activerecord-5.2.1/lib/active_record/log_subscriber.rb:98
Processing by Api::V1::PosicaoAgressorController#nova_posicao as */*
  Parameters: {"id"=>"1", "latitude"=>"-17.4568", "longitude"=>"-7.9865", "posicao_agressor"=>{"id"=>"1", "latitude"=>"-17.4568", "longitude"=>"-7.9865"}}
  [1m[36mAgressor Load (0.5ms)[0m  [1m[34mSELECT  "agressors".* FROM "agressors" WHERE "agressors"."id" = $1 LIMIT $2[0m  [["id", 1], ["LIMIT", 1]]
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:19
Unpermitted parameter: :posicao_agressor
Unpermitted parameter: :posicao_agressor
  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
  [1m[35m (0.2ms)[0m  [1m[35mCOMMIT[0m
  ‚Ü≥ app/controllers/api/v1/posicao_agressor_controller.rb:8
Completed 200 OK in 29ms (Views: 0.4ms | ActiveRecord: 6.4ms)


